docs/tutorial.md

```diff
@@ -6,26 +6,61 @@ sidebar_label: Your first Aragon app
 
 #####
 
-
-In this guide, we will walk you through creating your first Aragon app using [aragonOS](os-intro.md), the JavaScript implementation of [aragonAPI](api-intro.md), [aragonUI](ui-intro.md) and [aragonCLI](/docs/cli-intro.html).
+In this guide, we will walk you through creating your first Aragon app using [aragonOS](os-intro.md), the JavaScript implementation of [aragonAPI](api-intro.md), [aragonUI](ui-intro.md) and the Aragon [buidler plugin](https://github.com/aragon/buidler-aragon).
 
 ## The setup
 
 Let's first set up and bootstrap our project:
 
 ```sh
-npx create-aragon-app foo.aragonpm.eth tutorial
+npx create-aragon-app foo tutorial
 ```
 
-This will create a new directory named `foo`, with files cloned from [your first Aragon app template](https://github.com/aragon/your-first-aragon-app). This particular boilerplate includes everything you need to get started — Truffle, aragonOS and aragonAPI.
+This will create a new directory named `foo`, with files cloned from [your first Aragon app template](https://github.com/aragon/your-first-aragon-app). This particular boilerplate includes everything you need to get started — [buidler](https://buidler.dev/), aragonOS and aragonAPI.
+
+### Structure
+
+This boilerplate has the following structure:
+
+```md
+root
+├── app
+├ ├── src
+├ ├ ├── App.js
+├ ├ ├── index.js
+├ ├ └── script.js
+├ └── package.json
+├── contracts
+├ └── CounterApp.sol
+├── scripts
+├ └── buidler-hooks.js
+├── arapp.json
+├── manifest.json
+├── buidler.config.js
+└── package.json
+```
 
-Notice that we input a fully qualified [ENS](https://ens.domains/) name. Let's examine the ENS name we entered, because it is not entirely arbitrary.
+- **app**: Frontend folder. Completely encapsulated, has its own package.json and dependencies.
+  - **src**: Source files.
+    - `App.js`: Aragon app root component.
+    - `index.js`: Aragon app entry point.
+    - `script.sol`: Aragon app background script.
+  - [**package.json**](https://docs.npmjs.com/creating-a-package-json-file): Frontend npm configuration file.
+- **contracts**: Smart Constracts folder.
```

Comment:
> Smart **contracts** folder

We may want to double check the other boilerplate repos and see if we also need to update them there.